web-app/js/portal/lang/en.js

```diff
@@ -179,7 +179,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     resetActionText: 'reset',
 
     // FacetFilterPanel
-    addAnother: 'Add another filter',
+    addAnother: 'Add another filter - where possible',
```

Comment:
@dnahodil Agh yes 
If someone see's this tooltip, (when possible ! hahaa) they will then have an explanation as to why the link disappears sometimes.